lib/improver/utilities/cube_metadata.py

```diff
@@ -145,7 +182,7 @@ def add_coord(cube, coord_name, changes, warnings_on=False):
         bounds = changes['bounds']
     units = None
     if 'units' in changes:
-        units = changes['units']
+        units = changes["units"]["units"]
```

Comment:
I'm a bit confused by this change (it makes sense, but only in the context of https://github.com/MetOffice/monow_suite/pull/17/files).  Is it possible to put a bit more info in the docstring about the format of the `changes` dict please?

Reply:
This has been reverted.